man/man8/zfs.8

```diff
@@ -3035,6 +3037,40 @@ Print verbose information about the stream and the time required to perform the
 
 .sp
 .ne 2
+.mk
+.na
+\fB-o\fR \fIproperty\fR=\fIvalue\fR
+.ad
+.sp .6
+.RS 4n
+Sets the specified property as if the command \fBzfs set\fR \fIproperty\fR=\fIvalue\fR was invoked at the same time the received dataset was created from the non-incremental send stream or updated from the incremental send stream.
```

Comment:
You might add: When receiving a stream from `zfs send -R`, causes the property to be inherited by all descendant datasets, as through `zfs inherit property` was run on any descendant datasets that have this property set on the sending system.